README.rst

```diff
@@ -75,39 +75,39 @@ To see what coala can do for your language, run:
+    +============================+============================+============================+
+    | C                          | Latex                      | SQL                        |
+    +----------------------------+----------------------------+----------------------------+
+    | C++                        | Lua                        | TypeScript                 |
+    +----------------------------+----------------------------+----------------------------+
+    | C#                         | Markdown                   | VHDL                       |
+    +----------------------------+----------------------------+----------------------------+
+    | CMake                      | Matlab/Octave              | Vimscript                  |
+    +----------------------------+----------------------------+----------------------------+
+    | CoffeeScript               | Natural Language (English) | XML                        |
+    +----------------------------+----------------------------+----------------------------+
+    | CSS                        | Perl                       | YAML                       |
+    +----------------------------+----------------------------+----------------------------+
+    | Dart                       | PHP                        |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | Fortran                    | Python 2                   |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | Go                         | Python 3                   |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | Haskell                    | R                          |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | HTML                       | reStructured Text          |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | Java                       | Ruby                       |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | JavaScript                 | Scala                      |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | JSP                        | SCSS                       |                            |
+    +----------------------------+----------------------------+----------------------------+
+    | Julia                      | sh & bash scripts          |                            |
+    +----------------------------+----------------------------+----------------------------+
```

Comment:
@sils - I have no idea ... I definitely went in and took that indentation away and now it looks like it has a 2 space indentation instead of just the one that I had before.  I also definitely wrote that commit message with reference to the issue so I'm pushing something other than what I think I am.  I'll keep working on it until you get frustrated with me :)